catalog/dags/data_refresh/dag_factory.py

```diff
@@ -51,23 +51,27 @@
 from common.sql import PGExecuteQueryOperator
 from data_refresh.data_refresh_task_factory import create_data_refresh_task_group
 from data_refresh.data_refresh_types import DATA_REFRESH_CONFIGS, DataRefresh
+from data_refresh.recreate_view_data_task_factory import DROP_DB_VIEW_TASK_ID
+from data_refresh.recreate_view_data_task_factory import (
+    GROUP_ID as RECREATE_MATVIEW_GROUP_ID,
+)
+from data_refresh.recreate_view_data_task_factory import create_recreate_view_data_task
 from data_refresh.refresh_popularity_metrics_task_factory import (
     GROUP_ID as REFRESH_POPULARITY_METRICS_GROUP_ID,
 )
 from data_refresh.refresh_popularity_metrics_task_factory import (
     UPDATE_MEDIA_POPULARITY_METRICS_TASK_ID,
     create_refresh_popularity_metrics_task_group,
 )
-from data_refresh.refresh_view_data_task_factory import (
-    UPDATE_DB_VIEW_TASK_ID,
-    create_refresh_view_data_task,
-)
 from data_refresh.reporting import report_record_difference, report_status
 
 
 logger = logging.getLogger(__name__)
 
-REFRESH_MATERIALIZED_VIEW_TASK_ID = UPDATE_DB_VIEW_TASK_ID
+# The first task in the recreate_matview TaskGroup
+RECREATE_MATERIALIZED_VIEW_TASK_ID = (
+    f"{RECREATE_MATVIEW_GROUP_ID}" f".{DROP_DB_VIEW_TASK_ID}"
+)
```

Comment:
This looks a little wonky, I think black might have gotten the better of it?
```suggestion
RECREATE_MATERIALIZED_VIEW_TASK_ID = (
    f"{RECREATE_MATVIEW_GROUP_ID}.{DROP_DB_VIEW_TASK_ID}"
)
```

Reply:
That is very weird, yep I didn't notice it had changed the formatting. Thanks!